siren/main.go

```diff
@@ -24,7 +25,8 @@ func main() {
 		os.Exit(0)
 	}
 
-	c, err := NewMPD(*mpdURL)
+	u := url(*mpdURL)
```

Comment:
I'd expect the command line argument to override `MPD_HOST` if a command line argument is given. It doesn't seem to, though?

Reply:
fair enough!